support/cas-server-support-couchdb-core/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<Configuration shutdownHook="disable">
+    <Appenders>
+        <Console name="console" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d %p [%c] - &lt;%m&gt;%n"/>
+        </Console>
+        <RollingFile name="file" fileName="build/gauthcouchdb.log" append="true"
```

Comment:
Likely copy/paste error, for the file name.

Reply:
Done